src/applications/disability-benefits/4142/pages/facilityAddress.jsx

```diff
@@ -0,0 +1,182 @@
+import _ from 'lodash/fp';
+
+// Example of an imported schema:
+// import fullSchema4142 from '../../../../vets-json-schema/dist/21-4142-schema.json';
+// In a real app this would be imported from `vets-json-schema`:
+import fullSchema4142 from 'vets-json-schema/dist/21-4142-schema.json';
+
+// In a real app this would not be imported directly; instead the schema you
+// imported above would import and use these common definitions:
+import PrivateProviderTreatmentView from '../components/PrivateProviderTreatmentView';
+import {
+  schema as addressSchema,
+  uiSchema as addressUI,
+} from '../../../../platform/forms/definitions/address';
+
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+// import environment from '../../../../platform/utilities/environment';
+
+// import IntroductionPage from '../containers/IntroductionPage';
+// import ConfirmationPage from '../containers/ConfirmationPage';
```

Comment:
Same with these comments. If they're for reference, then I'd keep them, otherwise they can be removed.

Reply:
@robfelder : I actually didn't get to use this. I had to integrate within 526 and couldn't import this file. My question is: should I just delete the entire 4142 directory and merge 4142 helpers file into 526 directory?

Reply:
@igreulich : Would also like your input about this since you referenced helpers being in two files. My proposal would be to: 
1. Move all files actually being used to 526 directory: helpers, tests, and PrivateProviderTreatmentView.
2. delete entire 4142 directory
3. NOTE: I still need to update tests.

Reply:
I'm okay with moving the files to the 526 directory. Maybe myself or Ian can do this earlier in the day?